docs/index.rst

```diff
@@ -28,6 +26,19 @@ Useful links
 
 * `Gitter Chat <https://gitter.im/ethereum/solidity/>`_
 
+Available IDE
+-------------
+* `Ethereum Studio <https://live.ether.camp/>`_
+
+* `Browser-Based Compiler <https://chriseth.github.io/browser-solidity/>`_
+
+* `VS Extention <https://visualstudiogallery.msdn.microsoft.com/96221853-33c4-4531-bdd5-d2ea5acc4799/>`_
+
+* `Mix IDE <https://github.com/ethereum/mix/>`_
+
+* `MeteorJS development environment DApp <https://github.com/SilentCicero/meteor-dapp-cosmo/>`_
+
```

Comment:
There should also be extensions for atom, syntax hilighting for sublime and of course the Visual Studio Code extension.